lib/validic/version.rb

```diff
@@ -1,3 +1,3 @@
 module Validic
-  VERSION = '0.4.1'
+  VERSION = '0.5.0'
```

Comment:
we also need to add the `next` and `previous` methods to the README and explain the version bump